src/webrtc/stats/statsReportGatherer.ts

```diff
@@ -50,6 +48,7 @@ export class StatsReportGatherer {
 
     public async processStats(groupCallId: string, localUserId: string): Promise<SummaryStats> {
         const summary = {
+            isFirstCollection: this.previousStatsReport === undefined,
```

Comment:
Would it be beneficial to store the index of the stat report of the pc in here? this might be beneficial in the future anyways and filtering could happen with index == 0?

```suggestion
            summaryIndex: this.previousStatsReport === undefined ? 0 : this.previousStatsReport.summaryIndex + 1,
```